Makefile.am

```diff
@@ -157,10 +156,55 @@ mavlink_routerd_SOURCES = \
 	src/common/xtermios.cpp \
 	src/common/xtermios.h
 
+mavlink_routerd_CPPFLAGS = ${AM_CPPFLAGS}
+mavlink_routerd_CPPFLAGS += \
+	-I$(top_srcdir)/src/mavlink-router \
+	-I$(top_builddir)/include/mavlink \
+	-I$(top_builddir)/include/mavlink/ardupilotmega
+
+if RTPS
+bin_PROGRAMS += rtpsd
+rtpsd_SOURCES = \
+	src/common/log.cpp \
+	src/common/log.h \
+	src/common/util.c \
+	src/common/util.h \
+	src/rtps/main.cpp \
+	src/rtps/RtpsTopics.cxx \
```

Comment:
> The tool to generate these files should probably be extracted in a way that it can be used on both sides of the bridge.

Actually tool is generating code for both sides of bridge (client and agent). But we are not using micrortps_agent as is. We took couple of files from there and used in mavlink-router.
@edersondisouza any specific reason that we cant use micrortps_agent as is?